contracts/cw-ics721-bridge/src/ibc_packet_receive.rs

```diff
@@ -160,7 +160,13 @@ pub(crate) fn receive_ibc_packet(
         )
         .into_submessage(env.contract.address, receiver)?;
 
-    Ok(IbcReceiveResponse::default()
+    let mut response = IbcReceiveResponse::default();
+
+    if data.memo.is_some() {
+        response = response.add_attribute("ics721_memo", data.memo.unwrap());
+    }
```

Comment:
```
let response = if let Some(memo) = data.memo {
   Response::default().add_attribute("memo", memo)
} else {
  Response::default()
}
```
nice to avoid making variables mutable where we don't have to. mutability is viral so where possible i try to avoid it

Reply:
nice, need to get used to the let if let thing